packages/fluentui/react-icons-northstar/src/components/FluentOneDriveIcon.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { createSvgIcon } from '../utils/createSvgIcon';
+
+export const FluentOneDriveIcon = createSvgIcon({
```

Comment:
```suggestion
export const OneDriveColorIcon = createSvgIcon({
```

Should be this `OneDriveColorIcon`?

Reply:
looks like duplicate